labs/lab0/chapter0.md

```diff
@@ -97,13 +99,10 @@ We have two more commands worth mentioning. First off, let's mention `minishift
 
 ## Container Development Kit (CDK) Walkthrough
 
-When we started minishift, we launched the software tooling component of the CDK. As we said before, the CDK provides a lot of support for containerizing your applications, and the major software tool is minishift.
+When we started minishift, we launched the software tooling component of the CDK. As we said before, the CDK provides a lot of support for containerizing your applications. However, the major software tool is minishift.
```

Comment:
ending a paragraph with a sentence that starts with `However` and starting the next paragraph with `However`. Maybe change wording?